integration/test/agent_util.go

```diff
@@ -1,21 +1,20 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: MIT
 
-//go:build linux && integration
-// +build linux,integration
```

Comment:
Why are we removing this?

Reply:
Forgot to added it back because of the constraints with my mac and I cannot run the test. However, added back with the newest commit.